### PR TITLE
Adds storefront/theme step to the setup wizard

### LINKED ISSUE #13088: Themes step in Wizard
@jameskoster can you add more details here?

### PR DESCRIPTION
**Pull request by mikejolley**
Also includes a background install to get Storefront installed and
activated on demand.

To test, use a default theme and go to wp-admin/index.php?page=wc-setup
The theme step will be after payments!

Closes #13088

### CODE REVIEW

File: includes/class-wc-install.php [hunk 1]
```diff
@@ -1085,6 +1086,65 @@ public static function background_installer( $plugin_to_install_id, $plugin_to_i
...
+
+				switch_theme( $theme_slug );
+			} catch ( Exception $e ) {
+				WC_Admin_Notices::add_custom_notice(
+					$theme_slug . '_install_error',
+					sprintf(
+						__( '%1$s could not be installed (%2$s). <a href="%3$s">Please install it manually by clicking here.</a>', 'woocommerce' ),
+						$theme_slug,
+						$e->getMessage(),
+						esc_url( admin_url( 'http://local.wordpress.dev/wp-admin/update.php?action=install-theme&theme=' . $theme_slug . '&_wpnonce=' . wp_create_nonce( 'install-theme_' . $theme_slug ) ) )
```
Comment (claudiosanches) [inline includes/class-wc-install.php]: Missing remove tests first.

```php
admin_url( 'update.php....
```

**Review by claudiosanches — APPROVED** [review]
I just tested and looks good.
Missing remove a test from the code first, and after will be good to go.

**Review by jameskoster — COMMENTED** [review]
Headline copy is missing the first bit;

> Choose a theme to kickstart your store: Storefront is our free WordPress theme, built and maintained by the makers of WooCommerce.

Looks good otherwise.

**Comment by mikejolley** [comment]
@jameskoster I got the updated copy from @avivapinchas on slack.

@avivapinchas did you want to take a look at this before merging? It looks like this:

2017-05-18 at 10 39

**Pull request merged**